code/modules/materials/materials.dm

```diff
@@ -1011,6 +1011,8 @@ var/list/name_to_material
 	protectiveness = 3 // 13%
 	conductive = 0
 
+//CHOMPEDIT: Moved to materials_ch and changed to allow for material var

```

Comment:
I guess you want chompstation, thought CHOMP in comment was enough to indicate a comment, but I will do that in the future then if insisted
And I moved it all to materials_ch since there was to be a lot of changes so better to not deal with long chompedit start/end or numerous chompedit comments